examples/imagenet/sketch.js

```diff
@@ -27,7 +27,7 @@ function imageReady() {
   image(img, 0, 0, width, height);
 
   // Get a prediction for that image
-  imagenet.predict(img.elt, 10, gotResult);
+  imagenet.predict(img.elt, 10).then(gotResult);
```

Comment:
Can this be updated to the default callback? `imagenet.predict(img.elt, 10, gotResult);`

Reply:
Sure, I'll change that back!